src/Pages/SignUp/Identify.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import styled from 'styled-components';
+import Theme from '../../Styles/Theme';
+
+function Identify(props) {
+  return (
+    <Wrapper>
+      <Title>휴대폰 본인 확인</Title>
+      <d>원활한 서비스 제공을 위해, 휴대폰 번호를 입력해주세요.</d>
+
+      <Section>
+        <Phone>휴대폰 번호</Phone>
+        <InputDiv>
+          <Input type="text" maxLength="11" />
+          <Btn>인증번호 전송</Btn>
+        </InputDiv>
+
+        <Phone>인증번호</Phone>
+        <InputDiv>
+          <Input type="text" maxLength="4" />
+          <Btn type="submit">확인</Btn>
+        </InputDiv>
+      </Section>
+    </Wrapper>
+  );
+}
+
+export default Identify;
+
+const Wrapper = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 150px;
+  height: 100vh;
+  font-size: 20px;
+`;
+
+const Title = styled.div`
+  font-size: 34px;
+  font-weight: bold;
+  margin-bottom: 40px;
+`;
+
+const Section = styled.div`
+  margin: 40px;
+`;
+
+const Phone = styled.div`
+  margin-bottom: 10px;
+`;
+
+const InputDiv = styled.div`
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 30px;
+`;
+
+const Input = styled.input`
+  width: 310px;
+  height: 60px;
+  padding: 0 12px;
+  border: 1px solid ${Theme.borderGray};
```

Comment:
```suggestion
  border: 1px solid ${props => props.theme.borderGray};
```
theme 객체를 import 할 필요가 없습니다. themeProvider에 넣어주었으므로 위와 같이 어디서든 바로 꺼내어 쓸 수 있습니다.